core/src/main/java/de/jplag/merging/MergingParameters.java

```diff
@@ -2,17 +2,17 @@
 
 /**
  * Collection of parameters that describe how a match merging should be performed.
- * @param mergeBuffer describes how shorter a match can be than the Minimum Token Match (Defaults to 0).
- * @param seperatingThreshold describes how many tokens can be between to neighboring matches (Defaults to 0).
+ * @param neighborLength describes how short a match can be, to be considered (Defaults to 2).
+ * @param gapSize describes how many tokens can be between to neighboring matches (Defaults to 6).
  */
-public record MergingParameters(boolean enabled, int mergeBuffer, int seperatingThreshold) {
+public record MergingParameters(boolean enabled, int neighborLength, int gapSize) {
 
     /**
-     * The default values of MergingParameters are false for the enable-switch and 0 for both mergeBuffer and
-     * seperatingThreshold. These completely deactivate MatchMerging.
+     * The default values of MergingParameters are false for the enable-switch, which deactivate MatchMerging, while
+     * neighborLength and gapSize default to (2,6), which in testing yielded the best results.
      */
     public MergingParameters() {
```

Comment:
`MergingOptions` would be more in lien with the `ClusteringOptions`.

Reply:
Do you know an efficient way to rename the file, class and all references in eclipse?